### DISCUSSION TITLE
Request: a simple Flash Process - also for Bulk Flashing

### DISCUSSION THREAD

**Discussion by BuckligerBaron** [tbnobody/OpenDTU#221] [Q&A]
I flashed succsessful openDTU using VSCode and PlattformIO. Now some Balkonkraftwerk and GuerillaPV Communities asked me how they could also flash openDTU using my compiled Image in a simple Way. I have read the Docs but could not find any Hints. So here are my Questions:

1. where can i find the compiled Image after the Buld Process?
2. which specific Parameters must i use to flash openDTU via Commandline? Obviously these Informations are stored somewhere in a File inside the openDTU Folder which PlattformIO is using. But in which File?

Thanks in Advance for your Help to bring openDTU also to all Users

**Comment by ghost** [comment]
A look at "read.me" is always a good option.
In the chapter "Flashing and starting up" and the following ones you will find information on your questions.

Just as an aside: Your questions are not really an "Issue", in my opinion they would be better placed in the "Discussions" section.

**Reply by BuckligerBaron** [reply]
The Infos in the read.me were not clear enough for me. But in https://github.com/tbnobody/OpenDTU/discussions/208 User Madmartin made a easy to understand Description how to flash via Commandline. So this Discussion could be closed.
